### PR TITLE
Improve layout behavior

### PR DESCRIPTION
**Pull request by augustocb**
#### What problem is this solving?

It changes some paddings, margins and widths of the main elements to fix little issues in layout's behavior.

#### How should this be manually tested?

https://augusto--vtexgame1.myvtex.com/cart

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/styles.css [hunk 1]
```diff
@@ -1,5 +1,40 @@
+
+/* M */
 @media screen and (min-width: 40em) and (max-width: 63.9em) {
```
Comment (marcoskwkm) [inline react/styles.css]: @jgfidelis mentioned in a previous PR that `max-width` should be `64.1em` because of iPad Pro.

File: react/styles.css [hunk 2]
```diff
@@ -1,5 +1,40 @@
+
```
Comment (marcoskwkm) [inline react/styles.css]: Remove this empty line at the start of the file.

File: react/styles.css (hunk 1 above)
Comment (augustocb) [reply react/styles.css]: @davicosta99

Comment (davicosta99) [reply react/styles.css]: I don't think that it's unreasonable to assume that someone with a 12.9 tablet expects to have a desktop experience. Apple has even changed Safari in the last iOS version to deal with websites that weren't doing this.

**Comment by marcoskwkm** [comment]
Better rebase this branch before merging.

**Pull request merged**